example/pubspec.yaml

```diff
@@ -23,7 +23,10 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  flutter_map: ">=0.13.1 <1.0.0"
+  flutter_map: 
+    git: 
+      url: "https://github.com/sandeshgyw/flutter_map.git"
```

Comment:
Could you achieve the same result my using the released version of `flutter_map_location` but override the `flutter_map` using `depencency_override` in your app's `pubspec.yaml`?